lib/tests/streamlit/config_test.py

```diff
@@ -256,6 +256,7 @@ def test_config_option_keys(self):
             u'client.caching',
             u'client.displayEnabled',
             u'global.developmentMode',
+            u'global.disableWatchdogWarning',
```

Comment:
Nothing came to mind when I was developing.

Reply:
That would work, but you'd have to also:
1) Remove `LocalSourcesWatcher` from `sys.modules` before running the test, so the module gets imported at test time (in case it had previously been imported by another test)
2) Mock a bunch of things, like the function that tells you whether you're on a Mac